lang/cy.js

```diff
@@ -18,10 +18,10 @@ export default {
 	"components.filter.clearDescription": "Wrthi’n clirio hidlwyd ar gyfer: {filterName}",
 	"components.filter.clearDescriptionSingle": "Clirio Hidlwyr",
 	"components.filter.loading": "Wrthi’n llwytho hidlyddion",
-	"components.filter.filterCountDescription": "{number, plural, zero {Ni chymhwyswyd hidlwyr.} one {1 hidlydd wedi'i gymhwyso.} other {{number} hidlydd wedi'u cymhwyso.}}",
+	"components.filter.filterCountDescription": "{number, plural, =0 {Dim hildwyr wedi'u gweithredu.} un {1 hidlydd wedi'i weithredu.} arall {{number} hidlydd wedi'u gweithredu.}}",
```

Comment:
TODO: confirm if `zero` or `=0` should be used in Welsh. Also the options here got translated (one -> un, other -> arall) which is incorrect.

Reply:
Same for this one.  Both `zero` and `=0` return the same result from out `localize` function for `cy`.

Reply:
Old: "Ni chymhwyswyd hidlwyr." -> "Filters were not applied." ... makes sense to me.
New: "Dim hildwyr wedi'u gweithredu." -> "No surrenders implemented." ... 🤔 

Old: "1 hidlydd wedi'i gymhwyso." -> "1 filter applied."
New: "1 hidlydd wedi'i weithredu." -> "1 filter activated."

Old: "5 hidlydd wedi'u cymhwyso." -> "5 filters applied."
New: "5 hidlydd wedi'u gweithredu." -> "5 filters activated."

Not sure how much I trust our translators, and also not sure how much I trust Google translate.